modules/ducktests/tests/ignitetest/services/utils/templates/ignite.xml.j2

```diff
@@ -93,9 +95,41 @@
         {% if config.client_connector_configuration %}
             <property name="clientConnectorConfiguration">
                 <bean class="org.apache.ignite.configuration.ClientConnectorConfiguration">
-                        <property name="port" value="{{ config.client_connector_configuration.port }}"/>
+                    <property name="port" value="{{ config.client_connector_configuration.port }}"/>
+                    <property name="sslEnabled" value="{{ config.client_connector_configuration.ssl_enabled }}"/>
+                    {% if config.client_connector_configuration.ssl_enabled %}
```

Comment:
any reason to have `<property name="sslEnabled" value="false"` when !ssl_enabled?